src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php

```diff
@@ -73,7 +76,7 @@ public function __construct(
         ViewBuilderFactoryInterface $viewBuilderFactory,
         SecurityCheckerInterface $securityChecker,
         WebspaceManagerInterface $webspaceManager,
-        $defaultEnabled
+        $defaultEnabled,
```

Comment:
we don't have this in the sulu php-cs yet

```suggestion
        $defaultEnabled
```